Mathlib/LinearAlgebra/TensorProduct/RightExactness.lean

```diff
@@ -416,6 +417,58 @@ theorem TensorProduct.map_ker :
   rw [range_eq_top.mpr (rTensor_surjective M' hg), Submodule.map_top]
   rw [Exact.linearMap_ker_eq (lTensor_exact P hfg' hg')]
 
+variable (M)
+
+/-- Left tensoring a module with a quotient of the ring is the same as
+quotienting that module by the corresponding submodule. -/
+noncomputable def lTensor_ring_mod_ideal_equiv_mod_ideal_smul (I : Ideal R) :
+    ((R⧸I) ⊗[R] M) ≃ₗ[R] M⧸(I • (⊤ : Submodule R M)) :=
+  (rTensor.equiv M (exact_subtype_mkQ I) I.mkQ_surjective).symm.trans <|
+    Submodule.Quotient.equiv _ _ (TensorProduct.lid R M) <|
+      Eq.trans (LinearMap.range_comp _ _).symm <|
+        Eq.trans ((Submodule.topEquiv.lTensor I).range_comp _).symm <|
+          Eq.symm <| Eq.trans (map₂_eq_range_lift_comp_mapIncl _ _ _) <|
+            congrArg _ (TensorProduct.ext' (fun _ _ => rfl))
+
+/-- Right tensoring a module with a quotient of the ring is the same as
+quotienting that module by the corresponding submodule. -/
+noncomputable def rTensor_ring_mod_ideal_equiv_mod_ideal_smul (I : Ideal R) :
+    (M ⊗[R] (R⧸I)) ≃ₗ[R] M⧸(I • (⊤ : Submodule R M)) :=
+  TensorProduct.comm R M _ ≪≫ₗ lTensor_ring_mod_ideal_equiv_mod_ideal_smul M I
+
+variable {M}
+
+@[simp]
+lemma lTensor_ring_mod_ideal_equiv_mod_ideal_smul_apply
```

Comment:
Maybe `_mk_tmul` instead. `apply` is usually for plain function application.

Reply:
I don't think this is consistent? By searching for "_apply" in the docs I found [`L_apply`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Geometry/Manifold/Algebra/Monoid.html#L_apply), [`R_apply`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Geometry/Manifold/Algebra/Monoid.html#R_apply), [`starL_apply`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Module/Star.html#starL_apply), and [`innerₗ_apply`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/InnerProductSpace/Basic.html#inner%E2%82%97_apply) as counterexamples. See also a number of types with `apply` as a simps projection: https://github.com/search?q=repo%3Aleanprover-community%2Fmathlib4+initialize_simps_projections+%28toFun+%E2%86%92+apply%29&type=code.

Reply:
By plain function application I meant `⇑f x` where `x` is a variable. Here `x` is some complex term.